config/README.md

```diff
@@ -441,6 +441,57 @@ A web page can embed another web page with the `iframe` tag in HTML. You may cho
 </script>
 ```
 
+## Refresh Settings
+
+This setting is used to control Hyper SDK's refresh behavior.
+SDK will regularly check if the hyper card data get updated with this setting on.
```

Comment:
replace with:
"Hyper SDK will regularly check if the hyper card data has been updated when this setting is set to on."